<launch>
	<arg name="num_cops" default="1" />
	<arg name="rviz" default="false" />
	<node name="swarm_mavros" pkg="burrito" type="swarm_mavros.py" output="screen" args="$(arg num_cops)" />
	<node name="simulate_swarm" pkg="burrito" type="simulate_swarm.py" output="screen" args="$(arg num_cops)" />
	<node name="start_hellacopters" pkg="burrito" type="start_hellacopters.py" output="screen" args="$(arg num_cops)" />
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find burrito)/config/swarm_overhead.rviz" if="$(arg rviz)"/>
</launch>
